docs/preparation/index.md

```diff
@@ -39,6 +39,7 @@ Unfortunately, not all configuration can be done automatically, and some changes
   [`isInspectable`](https://developer.apple.com/documentation/webkit/wkwebview/4111163-isinspectable)
   property set to `true`. Please read [the WebKit documentation page](https://webkit.org/blog/13936/enabling-the-inspection-of-web-content-in-apps/)
   for more details on this property.
+  - Lower than 16.4 need to have [`get-task-allow` entitlement](https://developer.apple.com/documentation/bundleresources/entitlements/com_apple_security_cs_debugger) in the application under test.
```

Comment:
Webviews on iOS/iPadOS 16.4 below version 16.4 must have the ... present in the application manifest